### DISCUSSION TITLE
Integration with impress.js?

### DISCUSSION THREAD

**Discussion by sradhakrishna** [marp-team/marp#181] [Ideas]
Can you consider integrating impress.js? The 3d presentations are super cool!

**Comment by yhatt** [comment]
[Marpit framework](https://marpit.marp.app/) brought by Marp ecosystem exists for integrating with other presentation libraries! (Especially in [a planned version v3](https://github.com/marp-team/marpit/issues/194#issuecomment-557466206))

If communities have interested in integration with web slide frameworks other than Marp (Marp Core), anyone can create a customized engine from Marpit. See also: [Marpit + reveal.js](https://codesandbox.io/s/nw80vrxvpp), [Marpit + WebSlides](https://codesandbox.io/s/j3wo2091yw). The same approach also should work on impress.js.
